packages/Webkul/Product/src/Models/Product.php

```diff
@@ -225,6 +226,14 @@ public function customer_group_prices()
         return $this->hasMany(ProductCustomerGroupPriceProxy::modelClass());
     }
 
+    /**
+     * Get the catalog rule prices.
+     */
+    public function catalog_rule_prices()
```

Comment:
Remove this two-way relationship from here as we don't want the Product package to dependent on CatalogRule package